src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -447,18 +449,44 @@ var DynamicTable = new Class({
 
     setSortedColumn: function(column) {
         if (column != this.sortedColumn) {
+            var oldColumn = this.sortedColumn;
             this.sortedColumn = column;
             this.reverseSort = '0';
+            this.setSortedColumnIcon(column, oldColumn, false);
         }
         else {
             // Toggle sort order
             this.reverseSort = this.reverseSort == '0' ? '1' : '0';
+            this.setSortedColumnIcon(column, null, (this.reverseSort == '1'));
```

Comment:
since js is using dynamic typing, you can just pass `this.reverseSort` for the 3rd parameter, or do you prefer the current way?

Reply:
I wanted this call to be a little more explicit, so that `setSortedColumnIcon` was getting a boolean value.